src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs

```diff
@@ -857,11 +870,15 @@ private void Dispose(bool disposing)
         }
 
         // Unified method to read a value from the current row
-        private object GetValueFromSourceRow(int destRowIndex, out bool isSqlType, out bool isDataFeed, out bool isNull)
+        private Task ReadWriteValueGenericAsync(int destRowIndex)
```

Comment:
I can probably condense this method with `ReadWriteColumnValueAsync`, which ends up getting inlined anyway.